### ISSUE TITLE
group filters 

### ISSUE THREAD

**Issue by Sherman-Shank** [yanus171/Handy-News-Reader#326]
**Is your feature request related to a problem? Please describe.**

**Describe the solution you'd like**

**Describe alternatives you've considered**

**Additional context**

need a group filters. this is very useful thank single feed filter. 

i.e， I have a group for IT news, then I can set group filter to alert specific topic for all the feeds in these all IT news feeds.

**Comment by yanus171** [comment]
Why global is not enough?

**Comment by Sherman-Shank** [comment]
yes. currently I use global filter for this purpose. but it not perfect.


I  have a group for all deals. I use filter for those group try to find
cheap TV sale deal.  I set filter for "TV".

because it is globally, I get alerts from all the news feeds too. a little
bit annoying.

I'm a developer, I set filter "cloud" filter to get all about cloud news.
but I get alerts from other groups too.
